ops.go

```diff
@@ -566,12 +566,27 @@ func cmdListPackages(cmd *cobra.Command, args []string) {
 	table.Render()
 }
 
+func validateRunCmd(cmd *cobra.Command, args []string) error {
```

Comment:
Only, I would recommend parsing the argument array into a structure before validation since matching the raw argument positions is error prone and hard to test. Structured input validation, IMHO, would be the first step in eliminating the panics. Second would be handling those errors that bubble up to the top with a logging package this ways errors can be easily formatted for stdout and written to file.